test/prob/beta_neg_binomial/beta_neg_binomial_cdf_log_test.hpp

```diff
@@ -0,0 +1,92 @@
+// Arguments: Ints, Doubles, Doubles, Doubles
+#include <stan/math/prim/prob/beta_neg_binomial_lcdf.hpp>
+#include <stan/math/prim/fun/lbeta.hpp>
+#include <stan/math/prim/fun/lgamma.hpp>
+
+using stan::math::var;
+using std::numeric_limits;
+using std::vector;
+
+class AgradCdfLogBetaNegBinomial : public AgradCdfLogTest {
+ public:
+  void valid_values(vector<vector<double>>& parameters,
+                    vector<double>& ccdf_log) {
+    vector<double> param(4);
+
+    param[0] = 10;   // n
+    param[1] = 5.5;  // r
+    param[2] = 2.5;  // alpha
+    param[3] = 0.5;  // beta
+    parameters.push_back(param);
+    ccdf_log.push_back(std::log(0.967906252841089));  // expected cdf_log
```

Comment:
Similar to the lccdf, can you find values for this that don't take as long? This looks like another 1day+ test

Reply:
Yep, tested on the local computer, the required numbers are the same as in `lccdf`.